recipe/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2.2.3" %}
+{% set version = "3.0.0" %}
```

Comment:
Guessing this won't be merged until the rc part is gone?

Reply:
Exactly, I created the PR to make sure things are working before tagging the final release.